packages/domain-picker/src/components/domain-suggestion-item/__tests__/suggestion-item.tsx

```diff
@@ -118,6 +118,35 @@ describe( 'conditional elements', () => {
 		// use `queryBy` to avoid throwing an error with `getBy`
 		expect( screen.queryByTestId( 'info-tooltip' ) ).toBeFalsy();
 	} );
+
+	it.skip( 'renders info tooltip for domains that require .gay information notice', async () => {
```

Comment:
I'm guessing this is just a work-in-progress but we probably want to avoid adding skipped tests to the codebase.